holoviews/element/raster.py

```diff
@@ -408,45 +211,157 @@ class Image(SheetCoordinateSystem, Raster):
     access to the data, via the .data attribute.
     """
 
-    bounds = param.ClassSelector(class_=BoundingRegion, default=BoundingBox(), doc="""
+    bounds = param.ClassSelector(class_=BoundingRegion, default=None, doc="""
        The bounding region in sheet coordinates containing the data.""")
 
+    datatype = param.List(default=['image', 'grid', 'xarray', 'dataframe', 'dictionary'])
+
     group = param.String(default='Image', constant=True)
 
+    kdims = param.List(default=[Dimension('x'), Dimension('y')],
+                       bounds=(2, 2), constant=True, doc="""
+        The label of the x- and y-dimension of the Raster in form
+        of a string or dimension object.""")
+
     vdims = param.List(default=[Dimension('z')],
-                       bounds=(1, 1), doc="""
+                       bounds=(1, None), doc="""
```

Comment:
I think this should be ``bounds=(1,1)`` to be explicit - better than making an arbitrary choice about which vdim to use (e.g the first one).